lib/express.js

```diff
@@ -60,8 +89,8 @@ function createApplication() {
  */
 
 exports.application = proto;
-exports.request = req;
-exports.response = res;
+exports.request = req.proto
+exports.response = res.proto
```

Comment:
Yeah probably, hard to remember if I had even looked at that since it was so long ago, but I think you are right. I will hold off on addressing these because the other review comments are much more important for us to address first imo.